pom.xml

```diff
@@ -664,6 +708,10 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>sql-maven-plugin</artifactId>
+      </plugin>
```

Comment:
So this will execute also for IT tests, right? In that case we will execute "external" scripts in our test suite in the pre-integration phase and then in the integration-test phase of the invoker it will invoke this project, which will run the sql-maven-plugin again. Maybe we should skip this when running IT tests from the QE repository. Maybe when the `custom-container` profile is activated?

Reply:
Indeed, good catch. Changed as suggested, though I will use prod scripts to run both unit and integration tests